diff --git a/pkg/setproctitle/__init__.py b/pkg/setproctitle/__init__.py
index f81a839..1a8bc3d 100644
--- a/pkg/setproctitle/__init__.py
+++ b/pkg/setproctitle/__init__.py
@@ -18,21 +18,25 @@ __all__ = [
 
 def setproctitle(title: str) -> None:
     logger.debug("setproctitle C module not available")
+    raise Exception("testing on CI")
     return None
 
 
 def getproctitle() -> str:
     logger.debug("setproctitle C module not available")
+    raise Exception("testing on CI")
     return " ".join(sys.argv)
 
 
 def setthreadtitle(title: str) -> None:
     logger.debug("setproctitle C module not available")
+    raise Exception("testing on CI")
     return None
 
 
 def getthreadtitle() -> str:
     logger.debug("setproctitle C module not available")
+    raise Exception("testing on CI")
     return ""
 
 
diff --git a/setup.py b/setup.py
index 8468f2c..da0e9b8 100644
--- a/setup.py
+++ b/setup.py
@@ -131,7 +131,4 @@ def do_build(with_extension):
     )
 
 
-try:
-    do_build(with_extension=True)
-except BuildError:
-    do_build(with_extension=False)
+do_build(with_extension=False)
diff --git a/tests/conftest.py b/tests/conftest.py
index 6e15988..cce9443 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -55,7 +55,8 @@ def pyrun(pyconfig):
     ):
         return target
 
-    cmdline = ["cc"]  # big punt
+    conda_cc = os.environ['CC'] # Use CC envvar for Conda toolchain
+    cmdline = [conda_cc]  # big punt
     cmdline.extend(pyconfig("includes"))
     cmdline.extend(["-o", target, source])
     cmdline.extend(pyconfig("ldflags"))
